### PR TITLE
added moveIt method

### PR DESCRIPTION
**Pull request by vijeegovindan**
Added moveIt() method. Please review and comment. Thanks!

### CODE REVIEW

File: src/com/Processor.java [hunk 1]
```diff
@@ -30,6 +30,70 @@ public void processWorkOrders() {
...
+                        workOrderMap.put(Status.INITIAL, workOrderSetInitial);
+
+                    }else if(order.getStatus().equals( Status.INITIAL )){
+                        order.setStatus( Status.ASSIGNED);
+                        workOrderSetAssigned.add(order);
+                        workOrderMap.put(Status.ASSIGNED, workOrderSetAssigned);
+
+                    }else if(order.getStatus().equals( Status.ASSIGNED )){
+                        order.setStatus( Status.IN_PROGRESS);
+                        workOrderSetAssigned.add(order);
```
Comment (Kessident) [inline src/com/Processor.java]: I'm assuming you copy/pasted and missed this, should be workOrderSetInProgress

**Review by Kessident — COMMENTED** [review]
Other than that one typo, the logic there looks good, but I don't think moveIt() is dealing with files, just with the map contained in the Processor

File: src/com/Creator.java [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+
 
 public class Creator {
     public void createWorkOrders() {
         // read input, create work orders and write as json files
+
+        WorkOrder newWorkOrder = new WorkOrder();
+        Scanner input = new Scanner(System.in);
+
+        status = newWorkOrder.setStatus(Status.INITIAL);
```
Comment (Kessident) [inline src/com/Creator.java]: "status =" isn't needed

File: src/com/Creator.java [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+
+        System.out.println("Please enter your name.");
+        senderName = input.nextLine();
+
+        ObjectMapper mapper = newObjectMapper();
+        int workOrderIDNo = newWorkOrder.getId();
+        mapper.writeValue(new File(workOrderIDNo + ".json"),newWorkOrder);
+        String jsonString = mapper.writeValueAsString(newWorkOrder);
```
Comment (Kessident) [inline src/com/Creator.java]: This is just gonna make a string representing the JSON which isn't needed as the ling above is writing it to an actual file

File: src/com/Creator.java [hunk 4]
```diff
@@ -1,12 +1,39 @@
...
+        Scanner input = new Scanner(System.in);
+
+        status = newWorkOrder.setStatus(Status.INITIAL);
+
+        System.out.println("Please enter your name.");
+        senderName = input.nextLine();
+
+        ObjectMapper mapper = newObjectMapper();
+        int workOrderIDNo = newWorkOrder.getId();
+        mapper.writeValue(new File(workOrderIDNo + ".json"),newWorkOrder);
```
Comment (Kessident) [inline src/com/Creator.java]: This should be surrounded by a try/catch, as it might throw an IOException

File: src/com/Creator.java [hunk 5]
```diff
@@ -1,12 +1,39 @@
...
+        System.out.print("Your new order has been created.");
     }
 
     public static void main(String args[]) {
         Creator creator = new Creator();
         creator.createWorkOrders();
     }
 }
+
+    public static void main(String args[]) {
```
Comment (Kessident) [inline src/com/Creator.java]: there's two main methods

**Review by adrbin42 — APPROVED** [review]

**Pull request merged**